src/frontend/citizen-portal/src/app/components/find-ticket/find-ticket.component.ts

```diff
@@ -24,7 +24,7 @@ export class FindTicketComponent implements OnInit {
   dialogRef: MatDialogRef<WaitForOcrDialogComponent>;
   analyzingTicket: boolean = false;
   notFound = false;
-  toolTipData = 'If you cannot upload a copy of your handwritten ticket, you can initiate a dispute by contacting the Violation Ticket Centre: violationticketcentre@gov.bc.ca.  You must include your Given Name, Surname, Driver\'s Licence Number, Violation Ticket Number, and Violation Time. Do not include any evidence, such as videos or pictures, with your email. Evidence can only be presented at the hearing.';
+  toolTipData = 'If you cannot upload a copy of your handwritten ticket, please contact support at Courts.TCO@gov.bc.ca. You must include your Given Name, Surname and Violation Ticket Number.';
```

Comment:
@Sharkww , does it make sense to use `<a href="mailto:Courts.TCO@gov.bc.ca">Courts.TCO@gov.bc.ca</a>` here?

In other locations, we also include a mail icon, ie.
`<a href="mailto:Courts.TCO@gov.bc.ca" target="blank">Courts.TCO@gov.bc.ca <img class="me-2" src="/assets/mail-icon.svg" style="width: 15px; height: 13px; vertical-align: text-top;"/></a>`

Is there a standard or best practice?